eks/variables.tf

```diff
@@ -0,0 +1,39 @@
+#
+# Variables Configuration
+#
+
+variable "region" {
+    type = "string"
+}
+
+variable "workstation_cidr_block" {
+    type = "string"
+}
+
+variable "keypair_name" {
+    type = "string"
+}
+
+variable "eks_version" {
+    type = "string"
+}
+
+variable "cluster_labels" {
+    type = "map"
+}
+
+variable "hosted_zone_name" {
+    type = "string"
+}
```

Comment:
This is a variable in services module as well?

Reply:
This is declared for the user provided input value of `hosted_zone_id` from where services module infer its value:

`hosted_zone_name = "${var.hosted_zone_name}"`